apisix/discovery/nacos.lua

```diff
@@ -235,12 +237,12 @@ local function iter_and_add_service(services, values)
 
         local namespace_id
         if up.discovery_args then
-            namespace_id = up.discovery_args.namespace_id
+            namespace_id = up.discovery_args.namespace_id or default_namespace_id
```

Comment:
Look like the default value is not necessary here? We already use one in fetch_full_registry

Reply:
the data of `fetch_full_registry` is from nacos, and the data of `up.discovery_args` is from `conf`, so it's necessary here.